doitlive.py

```diff
@@ -181,6 +184,18 @@ class TTY(object):
     DIM = ANSICode(dim=True)
 
 
+def echo(message=None, file=None, nl=True, err=False, color=None, carriage_return=False):
```

Comment:
This wrapped `echo` function isn't part of doitlive's public API, so it doesn't need to have the same positional arguments as click's `echo`. It's more important that this wrapped function maintains compat with click>=4.0.